index.html

```diff
@@ -23,25 +23,162 @@
+
+    <ul class="social">
+        <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
+        <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
+         <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+        <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+        <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+    </ul>
+
+    </section>
+
+    <section  class="card" id="tonic">
+
+        <img src="images/portfolio.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Tonic</h1>
+
+        <img src="images/fame2.png" alt="frame" id="frame">
+
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
         </ul>
 
+        <h3 class="button"><a href="" id="see_project">See Project</a></h3>
+
     </section>
+
+    <section id="multi" class="card">
+
+        <img src="images/portfolio2.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Multi-Post Stories</h1>
+        <img src="images/fame2.png" alt="frame" id="frame">
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
+    </section>
+
+    <section id="tonic" class="card">
+
+        <img src="images/portfolio3.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Tonic</h1>
+        <img src="images/fame2.png" alt="frame" id="frame">
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
```

Comment:
- [ ] Good you use an `a` tag for the "See project" button but why are you wrapping an `h3` tag here? Please remember that the `h3` tag is a heading tag and use for page heading so we don't use it for buttons. :smile: 
- [ ] Please follow the button styling: Active, Hover, and Pressed (see design)
  
  ![image](https://user-images.githubusercontent.com/92303891/235553643-733ea4a8-22ac-47f4-a305-89b6510743f6.png)